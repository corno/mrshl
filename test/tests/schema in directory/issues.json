[
    [ "expected '(' but found '{'", "warning", 1, 1, 1, 2],
    [ "unexpected property: 'a'", "error", 2, 5, 2, 8],
    [ "expected 'number' but found 'string'", "error", 3, 10, 3, 13],
    [ "expected ')' but found '}'", "warning", 5, 1, 5, 2],
    [ "missing property: 'a1'", "error", 1, 1, 1, 2]
]