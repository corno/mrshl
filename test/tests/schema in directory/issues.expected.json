[
    [ "deserialization", "warning", "expected a verbose type: ( )", 1, 1, 1, 2],
    [ "deserialization", "error", "unexpected property: 'a'. Choose from 'a24', 'b', 'c'", 5, 5, 5, 8]
]